firefox/ScreenSharing/index.js

```diff
@@ -0,0 +1,41 @@
+var pageMod = require("sdk/page-mod");
+var gDomains = ['*.example.com'];
```

Comment:
This is an unused variable because it is shadowed by other declarations of `gDomains` below. (Please enable linting?)